flant/negentropy
# Complex e2e tests

add tests for running system: 
1) kafka etc
2) vault (flant_iam + flant_iam_auth)
3) client (authd+cli)
4) server (authd+server-accessd)

Tests 
1) create tenant, project, user etc at vault
2) check binary exists at client and server
3) configure client and server (configs for authd etc)
4) run authd and server-access at server and check appearance of valid authorized_keys
